tools/mason/MasonHelp.chpl

```diff
@@ -590,7 +590,8 @@ proc masonPublishHelp(){
   writeln('    --dry-run                    Check to see if package is ready to be published');
   writeln('    --check                      Runs check to see if package can be published successfully to <registry>');
   writeln('    --ci-check                   Same as --check, except omits git origin checks');
-  writeln('    --[no-]update                [Do not] Prevent registries from being updated when a package is published.');
+  writeln('    --[no-]update                [Do not] Prevent registries from being updated when a package is published');
+  writeln('    --refresh-licenses           Force update the list of valid license names from the SPDX license list repo and exit');
```

Comment:
Might be good to somehow clarify that this does not run the normal publishing process.